tailwind.config.cjs

```diff
@@ -15,35 +15,35 @@ module.exports = {
       // Undefined colors will be chosen by daisyUI automatically.
       {
         pendulum: {
-          primary: '#907EA0',
-          'primary-content': '#ffffff',
-          secondary: '#F4F5F6',
-          'secondary-content': '#58667E',
-          accent: '#1DE7DF',
-          neutral: '#EFF2F5',
-          'base-100': '#fbfcfe',
-          'base-200': '#ffffff',
-          'base-content': '#58667E',
+            primary: '#907EA0',
+            'primary-content': '#ffffff',
+            secondary: '#F4F5F6',
+            'secondary-content': '#58667E',
+            accent: '#1DE7DF',
+            neutral: '#EFF2F5',
+            'base-100': '#fbfcfe',
+            'base-200': '#ffffff',
+            'base-content': '#58667E',
 
-          '--rounded-btn': '9px',
-          '--btn-text-case': 'none',
+            '--rounded-btn': '9px',
+            '--btn-text-case': 'none',
         },
         amplitude: {
-          primary: '#4EE59A',
-          'primary-content': '#ffffff',
-          secondary: '#F4F5F6',
-          'secondary-content': '#58667E',
-          accent: '#00F197',
-          'accent-content': '#ffffff',
-          neutral: '#191D24',
-          'neutral-focus': '#111318',
-          'neutral-content': '#A6ADBB',
-          'base-100': '#202020',
-          'base-200': '#1C1C1C',
-          'base-content': '#FFFFFF',
+            primary: '#4EE59A',
+            'primary-content': '#ffffff',
+            secondary: '#F4F5F6',
+            'secondary-content': '#58667E',
+            accent: '#00F197',
+            'accent-content': '#ffffff',
+            neutral: '#191D24',
+            'neutral-focus': '#111318',
+            'neutral-content': '#A6ADBB',
+            'base-100': '#1c1c1c',
+            'base-200': '#202020',
```

Comment:
This is tricky because it has indications on the other dialogs/cards as well. If you now look at the Portfolio Balance on the dashboard for example its background now has the same color as the background. 
Maybe we should keep it as it was before and add other global colors to the root css file.

Reply:
Hmm yeah, I mean I think this is the logical way based on the designs, I can change the portfolio then. Will it conflict with the branches you are working on?

Reply:
True. Apparently `base-100` should be the lightest color and `base-200` should be a darker shade of that. So it's correct how you changed it.

I'll check with the spacewalk ui once this is merged, maybe some changes are necessary but it should not be too much

Reply:
I reverted it based on your comment and fixed other places.